src/components/FormTitle/FormTitle.scss

```diff
@@ -0,0 +1,16 @@
+@import "./../../globalStyles/variables.scss";
+
+.formTitle {
+    display: flex;
+    background-color: $neutralPaper;
+
+    &__text {
+        font-family: $barlow;
+        font-size: 24px;
```

Comment:
Do we not have font size variables in the design system?
If we do you're goona need to carry em over.